test/libsolidity/semanticTests/enums/enum_explicit_overflow.sol

```diff
@@ -22,12 +22,13 @@ contract test {
 }
 
 // ====
-// EVMVersion: >=byzantium
 // compileViaYul: also
+// compileToEwasm: also
+// EVMVersion: >=byzantium
 // ----
 // getChoiceExp(uint256): 2 -> 2
-// getChoiceExp(uint256): 3 -> FAILURE, hex"4e487b71", 33 # These should throw #
-// getChoiceFromSigned(int256): -1 -> FAILURE, hex"4e487b71", 33
-// getChoiceFromMax() -> FAILURE, hex"4e487b71", 33
+// getChoiceExp(uint256): 3 -> FAILURE, hex"4e487b71", 0x21 # These should throw #
+// getChoiceFromSigned(int256): -1 -> FAILURE, hex"4e487b71", 0x21
+// getChoiceFromMax() -> FAILURE, hex"4e487b71", 0x21
```

Comment:
@cameel you asked about this change in another PR, and since this is a panic code IIRC, the hex version makes more sense